infra/repo_manager_test.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2019 Google LLC
```

Comment:
Maybe add "integration" to the names of this file and class because these are integration tests.